packages/amplify-provider-awscloudformation/lib/build-resources.js

```diff
@@ -28,11 +33,14 @@ async function buildResource(context, resource) {
     !resource.lastBuildTimeStamp ||
     new Date(packageJsonMeta.mtime) > new Date(resource.lastBuildTimeStamp)
   ) {
-    const npm = /^win/.test(process.platform) ? 'npm.cmd' : 'npm';
-    require('child_process').spawnSync(npm, ['install'], { cwd: resourceDir });
+    childProcess.spawnSync(npm, ['install'], { cwd: resourceDir });
```

Comment:
It would be nice to encapsulate the process spawn into a function and handle non-success results of running . NPM. In case of an error I think the command should fail and should not update the meta file and run a script. The same function could be reused below.

Reply:
Done.